drivers/amazonec2/stub_test.go

```diff
@@ -89,6 +91,35 @@ func (f *fakeEC2WithLogin) DescribeAccountAttributes(input *ec2.DescribeAccountA
 	}, nil
 }
 
+type fakeEC2SecurityGroupTestRecorder struct {
+	*fakeEC2
+	mock.Mock
+}
+
+func (f *fakeEC2SecurityGroupTestRecorder) DescribeSecurityGroups(
+	input *ec2.DescribeSecurityGroupsInput) (*ec2.DescribeSecurityGroupsOutput, error) {
+
+	result := f.Called(input)
+	value, _ := result.Get(0).(*ec2.DescribeSecurityGroupsOutput)
+	return value, result.Error(1)
+}
+
+func (f *fakeEC2SecurityGroupTestRecorder) CreateSecurityGroup(
+	input *ec2.CreateSecurityGroupInput) (*ec2.CreateSecurityGroupOutput, error) {
+
+	result := f.Called(input)
+	value, _ := result.Get(0).(*ec2.CreateSecurityGroupOutput)
+	return value, result.Error(1)
+}
+
+func (f *fakeEC2SecurityGroupTestRecorder) AuthorizeSecurityGroupIngress(
+	input *ec2.AuthorizeSecurityGroupIngressInput) (*ec2.AuthorizeSecurityGroupIngressOutput, error) {
```

Comment:
As discussed before, my preference would be to have this method definition on one line